internal/handler/name_handler.go

```
@@ -0,0 +1,49 @@
+
+    h.repo.CreateName(req.Name)
+    w.WriteHeader(http.StatusCreated)
+}
+
+// GetNames는 저장된 모든 이름을 조회하는 HTTP 엔드포인트 핸들러입니다.
+// 저장된 이름 목록을 JSON 형식으로 반환합니다.
+// TODO: success message 반환
+// TODO: 공백 문자열 반환
+func (h *NameHandler) GetNames(w http.ResponseWriter, r *http.Request) {
+    names := h.repo.GetNames()
+    json.NewEncoder(w).Encode(names)
```

golangci-lint (1.64.8)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)